### PR TITLE
Automation cli with TOML Config

### PR DESCRIPTION
**Pull request by cmalec**

### CODE REVIEW

**Review by EasterTheBunny — DISMISSED** [review]

**Review by anirudhwarrier — DISMISSED** [review]

File: core/scripts/chaincli/handler/handler.go [hunk 1]
```diff
@@ -47,6 +47,33 @@ const (
 	csaKeysEndpoint              = "/v2/keys/csa"
 )
 
+const (
```
Comment (ferglor) [inline core/scripts/chaincli/handler/handler.go:50]: I don't expect you to change this as part of this PR (I think I'm just spitballing this idea and I'm happy to make changes myself!)

But I was wondering if it made sense to use this library for generating toml configs:

https://github.com/BurntSushi/toml

So we could define everything in structs and use `toml` tags for the fields, and then use a toml encoder to generate the actual toml files?

**Review by ferglor — COMMENTED** [review]
Can we delete the commented out blocks? Or are they going to be introduced again at a later time?

File: core/scripts/chaincli/handler/report.go [hunk 2]
```diff
@@ -322,69 +319,69 @@ func (t *OCR2TransmitTx) UpkeepsInTransmit() ([]plugintypes.UpkeepResult, error)
...
+//		Args: []interface{}{
+//			map[string]interface{}{
+//				"from": from.Hex(),
+//				"to":   t.To().Hex(),
+//				"data": hexutil.Bytes(t.tx.Data()),
+//			},
+//			hexutil.EncodeBig(big.NewInt(int64(bn) - 1)),
+//		},
+//	}, nil
+//}
```
Comment (RyanRHall) [inline core/scripts/chaincli/handler/report.go:384]: do we need these functions? if not let's :scissors: them

Comment (cmalec) [reply core/scripts/chaincli/handler/report.go]: fixed

**Review by FelixFan1992 — APPROVED** [review]

**Pull request merged**